modules/deploy/pages/deployment-option/self-hosted/kubernetes/k-deploy-connectors.adoc

```diff
@@ -0,0 +1,749 @@
+----
+
+Here, `_internal_connectors_offsets` is the dedicated Kafka topic where Kafka Connect persists the offsets of the source connector.
+
+For all available settings, see the xref:reference:connector-helm-spec.adoc#connectors[Helm specification].
+
+== Verify the deployment
+
+To verify that the deployment was successful, you can use either Redpanda Console or the Kafka Connect REST API:
+
+=== Verify in Redpanda Console
+
+. Expose Redpanda Console to your localhost:
++
+[,bash]
+----
+kubectl --namespace <namespace> port-forward svc/redpanda-console 8080:8080
+----
++
+This command actively runs in the command-line window. To execute other commands while the command is running, open another command-line window.
+
+. Open Redpanda Console on http://localhost:8080.
+
+. Go to *Connectors*.
+
+You should see:
+
+- A list of available connectors (types)
+- The address of your Kafka Connect cluster
+- The version of Kafka Connect that you are running
+
+From here, you can create and configure instances of your connectors.
+
+=== Verify with the Kafka Connect REST API
+
+. Get the name of the Pod that's running Redpanda Connectors:
++
+[,bash]
+----
+kubectl get pod -l app.kubernetes.io/name=connectors --namespace <namespace>
+----
+
+. View the version of Kafka Connect:
++
+[,bash]
+----
+kubectl exec <pod-name> --namespace <namespace> -- curl localhost:8083 | jq
+----
++
+.Example output
+[%collapsible]
+====
+[,json,.no-copy]
+----
+{
+  "version": "3.5.1",
+  "commit": "2c6fb6c54472e90a",
+  "kafka_cluster_id": "redpanda.58d6bd99-7f7c-4732-a398-b44bf892979a"
+}
+----
+====
+
+. View the list of available connectors:
++
+[,bash]
+----
+curl localhost:8083/connector-plugins | jq
+----
++
+.Example output
+[%collapsible]
+====
+[,json,.no-copy]
+----
+[
+  {
+    "class": "org.apache.kafka.connect.mirror.MirrorCheckpointConnector",
+    "type": "source",
+    "version": "3.5.1"
+  },
+  {
+    "class": "org.apache.kafka.connect.mirror.MirrorHeartbeatConnector",
+    "type": "source",
+    "version": "3.5.1"
+  },
+  {
+    "class": "org.apache.kafka.connect.mirror.MirrorSourceConnector",
+    "type": "source",
+    "version": "3.5.1"
+  }
+]
+----
+====
+
+== Install a new connector
+
+To install new connectors other than the ones included in the Redpanda Connectors Docker image, you must:
+
+. Prepare a JAR (Java archive) file for the connector.
+. Mount the JAR file into the plugin directory of the Redpanda Connectors Docker image.
+. Use that Docker image in the Helm chart.
+
+=== Prepare a JAR file
+
+Kafka Connect is written in Java. As such, connectors are also written in Java and packaged into JAR files. JAR files are used to distribute Java classes and associated metadata and resources in a single file. You can get JAR files for connectors in many ways, including:
+
+- Build from source: If you have the source code for a Java project, you can compile and package it into a JAR using build tools like:
```

Comment:
Have you considered making the "Build from source" boldface with a period? Easier to scan and see each of the options here/below.